### PR TITLE
{bio}[foss,2016a] vcfnp v2.3.0

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ('Python', '2.7.11'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/vcf2npy'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
+
+sanity_check_commands = [
+    ('python', "-c 'import vcfnp'"),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0-foss-2016a-Python-2.7.11.eb]: unnecessary, this is handled by the easyblock

File: easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0_numpy_setup.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
+diff -ru vcfnp-2.3.0/setup.py vcfnp-2.3.0.fixed/setup.py
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0_numpy_setup.patch]: please add a descriptive comment including author

Comment (migueldiascosta) [reply easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0_numpy_setup.patch]: you'll also need to update the patch checksum :)

Comment (timeu) [reply easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0_numpy_setup.patch]: ups yes, fixed it now

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/e0ea68d35e479bfd423ff07780a7e1b2 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @timeu!

**Pull request merged**